pkg/client/parse.go

```diff
@@ -166,3 +167,10 @@ func getEndTime(startTime time.Time, durationSeconds string) (endTime time.Time,
 	endTime = startTime.Add(ms)
 	return
 }
+
+func convertToTags(tagString string) (tags []fern.Tag) {
```

Comment:
This is implicitly tested by other functions, but we should probably add a small unit test for this function anyway.

Reply:
added unit test for `convertToTags`